web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -188,6 +198,10 @@ export default class AccessGrants extends Vue {
      */
     private isDeleteClicked = false;
 
+    private showModal = false;
+    private modalClickType = 'Access Grant';
+    private modalPropType = '';
```

Comment:
bad name or add comments

Reply:
resolved
`    
     /**
     * Indicates if the access modal should be shown and what the defaulted type of access should be defaulted.
     */
    private showAccessModal = false;
    private modalAccessType = '';
`